### PR TITLE
UWSHIEVMOD-73: validate related websites url format

### PR DESCRIPTION
**Pull request by Tao-Liu**

### CODE REVIEW

File: app/models/data_file.rb [hunk 1]
```diff
@@ -180,6 +182,15 @@ def related_website_list
     self.related_websites.collect(&:url).join("|")
   end
 
+  def validate_related_websites
+    related_websites = related_website_list.split("|")
```
Comment (dyazbek-intersect) [inline app/models/data_file.rb]: Shouldn't you split on space?

**Comment by dyazbek-intersect** [comment]
Please write a cucumber test for this functionality.

File: app/models/data_file.rb (hunk 1 above)
Comment (Tao-Liu) [inline app/models/data_file.rb]: The existing code split related websites on '|' so I follow that. Should I refactor code for related websites?

Comment (Tao-Liu) [inline app/models/data_file.rb]: The existing code split related websites on '|' so I follow that. Should I
refactor code for related websites?

Comment (dyazbek-intersect) [inline app/models/data_file.rb]: No - I think its the select2 field that separates them with pipes. All good..

**Comment by Tao-Liu** [comment]
Refactored validation method and added cucumber test for it. Please review again.

**Comment by dyazbek-intersect** [comment]
nice :+1:

**Pull request merged**